### DISCUSSION TITLE
Raylib now on MSYS2!!!

### DISCUSSION THREAD

**Discussion by jacknicklenson** [raysan5/raylib#2448] [General]
Hi,

I asked MSYS2 maintainers to add raylib package to their repo and they' ve added. Therefore, windows users can install raylib as a dependency for their project easily with that command in mingw shell `pacman -S mingw-w64-x86_64-raylib`.

MSYS2 project -> https://msys2.org
MSYS2 raylib   -> https://packages.msys2.org/package/mingw-w64-x86_64-raylib?repo=mingw64

**Comment by raysan5** [comment]
@jacknicklenson Wow! That's amazing! :D

**Reply by jacknicklenson** [reply]
thx for great library :D

**Comment by Xotchkass** [comment]
The only problem is that they for some reason install it as a [dll instead of static library](https://github.com/msys2/MINGW-packages/blob/a7bb0638bed896a2cc134f8a84eb3e004aaa1207/mingw-w64-raylib/PKGBUILD#LL37C29-L37C29). 
```shell
ldd .\out\main.exe
        ntdll.dll => /c/Windows/SYSTEM32/ntdll.dll (0x7ffe54790000)
        KERNEL32.DLL => /c/Windows/System32/KERNEL32.DLL (0x7ffe53cd0000)
        KERNELBASE.dll => /c/Windows/System32/KERNELBASE.dll (0x7ffe520d0000)
        msvcrt.dll => /c/Windows/System32/msvcrt.dll (0x7ffe529c0000)
        libraylib.dll => /mingw64/bin/libraylib.dll (0x7ffdfa970000)
        GDI32.dll => /c/Windows/System32/GDI32.dll (0x7ffe53a10000)
        win32u.dll => /c/Windows/System32/win32u.dll (0x7ffe51f90000)
        gdi32full.dll => /c/Windows/System32/gdi32full.dll (0x7ffe51fc0000)
        msvcp_win.dll => /c/Windows/System32/msvcp_win.dll (0x7ffe523a0000)
        ucrtbase.dll => /c/Windows/System32/ucrtbase.dll (0x7ffe52580000)
        USER32.dll => /c/Windows/System32/USER32.dll (0x7ffe53870000)
        SHELL32.dll => /c/Windows/System32/SHELL32.dll (0x7ffe54000000)
        WINMM.dll => /c/Windows/SYSTEM32/WINMM.dll (0x7ffe4d910000)
        libgcc_s_seh-1.dll => /c/Program Files/Qalculate/libgcc_s_seh-1.dll (0x7ffe2b680000)  
        libwinpthread-1.dll => /c/Program Files/Qalculate/libwinpthread-1.dll (0x7ffe31030000)
```
(Also, wth is with it depending on qalc?)

**Reply by raysan5** [reply]
It seems there is some missing build configuration, some libraries could be statically linked or just avoided. Afair, the only Windows dependencies should be:
```
KERNEL32.DLL
USER32.DLL
GDI32.DLL
SHELL32.dll
WINMM.dll
```
In any case, I see most of those libraries are related...

**Comment by michael-collard** [comment]
Thank's a lot guy to added packman package yesterday you're the best ! 💯
